src/quarterback/api/document.ts

```diff
@@ -99,23 +98,23 @@ export const listenStepUpdates = (
   ) => void,
   authToken: string
 ) => {
-  const listener = (event: EventSourceMessage) => {
-    if (event.data) {
-      const data = JSON.parse(event.data)
-      if (
-        typeof data.version != 'undefined' &&
-        data.steps &&
-        Array.isArray(data.steps) &&
-        data.clientIDs
-      ) {
-        dataListener(data.version, data.steps, data.clientIDs)
-      }
+  const listener = (event: MessageEvent) => {
+    const data = JSON.parse(event.data)
+    if (
+      typeof data.version != 'undefined' &&
+      data.steps &&
+      Array.isArray(data.steps) &&
+      data.clientIDs
+    ) {
+      dataListener(data.version, data.steps, data.clientIDs)
     }
   }
-
+  //host to your websocket server locally, but to manuscripts api in production
   listen(
-    `doc/${projectID}/manuscript/${manuscriptID}/listen`,
+    `ws://localhost:35553/api/v2/doc/${projectID}/manuscript/${manuscriptID}/listen`,
```

Comment:
I'm still not quite sure how this would work if we sent this straight to the api :thinking:

Reply:
Actually it should theoretically work fine